app/code/Magento/Authorization/Model/Rules.php

```diff
@@ -19,6 +19,7 @@
  * @method \Magento\Authorization\Model\Rules setAssertId(int $value)
  * @method string getPermission()
  * @method \Magento\Authorization\Model\Rules setPermission(string $value)
+ * @method setResources(array $resource)
```

Comment:
Please add return type here and add corresponding getter in form `int[]` or something - as specific as possible.

As always, please do amend commit and force push.